Makefile

```
@@ -7,7 +7,7 @@ GRADLEW := ./gradlew
 NULL := /dev/null
 endif
 
-.PHONY: install-cli check-bb build test format format-check delta iter coverage-summary bump-version release test-release
+.PHONY: install-cli check-bb build test format format-check delta iter coverage-summary bump-version release test-release class-size-mine
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)